tests/integration/test_main.py

```diff
@@ -325,6 +325,17 @@ def test_request_list__no_repo_slug__https(self, capsys, caplog):
         assert out ==  '  1\tdesc1                                                       \thttp://request/1\n  2\tdesc2                                                       \thttp://request/2\n  3\tdesc3                                                       \thttp://request/3\n'
         assert 'id' in caplog.text and 'title' in caplog.text and 'URL' in caplog.text
 
+    def test_request_list__no_repo_slug__https_dot_git(self, capsys, caplog):
+        from subprocess import call
+        call(['git', 'init', '-q', self.tempdir.name])
+        call(['git', '--git-dir={}/.git'.format(self.tempdir.name), 'remote', 'add', 'github', 'https://github.com/guyzmo/.git-repo'])
+        repo_slug, seen_args = self.main_request_list(rc=0, args={})
+        assert ('guyzmo', '.git-repo') == repo_slug
+        assert dict() == seen_args
+        out, err = capsys.readouterr()
+        assert out ==  '  1\tdesc1                                                       \thttp://request/1\n  2\tdesc2                                                       \thttp://request/2\n  3\tdesc3                                                       \thttp://request/3\n'
+        assert 'id' in caplog.text and 'title' in caplog.text and 'URL' in caplog.text
```

Comment:
The three lines above (335-337) are not really useful to validate the test, as it's covered by the surrounding tests.
What's important really is the assert on line 334.

So I'd be for removing those ☺